Resources/public/js/views/dashboard/plugins/ez-base-dashboardblockviewplugin.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-base-dashboardblockviewplugin', function (Y) {
+    'use strict';
+
+    /**
+     * Provides the dashboard base block view plugin class
+     *
+     * @module ez-base-dashboardblockviewplugin
+     */
+    Y.namespace('eZ.Plugin');
+
+    /**
+     * The dashboard base block view plugin
+     *
+     * @namespace eZ.Plugin
+     * @class DashboardBlockBaseView
+     * @constructor
+     * @extends Base
+     */
+    Y.eZ.Plugin.DashboardBlockBaseView = Y.Base.create('baseDashboardBlockViewPlugin', Y.Base, [], {
```

Comment:
The reason I created this plugin is to show how the block can be added to dashboard by third-party developers.
This way, I provide the way to add new blocks to dashboard easily. Third-party developers will just extend this plugin and set different `block` attribute value to contain the block view instance they want to add.
So this is the extension point.